config/clb-consumer/contact-data/delta-context-config.js

```diff
@@ -0,0 +1,256 @@
+              ?siteOne org:siteAddress ${subject}.
+            }
+            BIND(<http://mu.semte.ch/graphs/administrative-unit> AS ?graphOne)
+          }
+          OPTIONAL {
+            GRAPH <http://mu.semte.ch/graphs/worship-service> {
+              ?siteTwo org:siteAddress ${subject}.
+            }
+            BIND(<http://mu.semte.ch/graphs/worship-service> AS ?graphTwo)
+          }
+        }`
+    },
+    {
+      trigger: { // subjectType or predicateValue
+        subjectType: "locn:Address"
+      },
+      queryTemplate: (subject) => `
+        ${PREFIXES}
+        CONSTRUCT {
+          ${subject} ext:goesInGraph ?graphOne.
+          ${subject} ext:goesInGraph ?graphTwo.
+        } WHERE {
+          OPTIONAL {
+            GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+              ?siteOne organisatie:bestaatUit ${subject}.
+            }
+            BIND(<http://mu.semte.ch/graphs/administrative-unit> AS ?graphOne)
+          }
+          OPTIONAL {
+            GRAPH <http://mu.semte.ch/graphs/worship-service> {
+              ?siteTwo organisatie:bestaatUit ${subject}.
+            }
+            BIND(<http://mu.semte.ch/graphs/worship-service> AS ?graphTwo)
+          }
+        }`
+    },
+    //Predicate triggers to solve out of order deltas
+    {
+      trigger: { // subjectType or predicateValue
+        predicateValue: "org:hasSite"
+      },
+      queryTemplate: (subject) => `
+        ${PREFIXES}
+        CONSTRUCT {
+          ${subject} ext:contextDataGoesInGraph ?graphOne.
+          ${subject} ext:contextDataGoesInGraph ?graphTwo.
+          ?site ?pSite ?oSite.
+          ?contact ?pContact ?oContact.
+          ?address ?pAddress ?oAddress.
+          ?addressContact ?pAddressContact ?oAddressContact.
+        } WHERE {
+          OPTIONAL {
+            GRAPH <http://mu.semte.ch/graphs/administrative-unit> {
+              ${subject} a ?type.
+            }
+            BIND(<http://mu.semte.ch/graphs/administrative-unit> AS ?graphOne)
+          }
+          OPTIONAL {
+            GRAPH <http://mu.semte.ch/graphs/worship-service> {
+              ${subject} a ?type.
+            }
+            BIND(<http://mu.semte.ch/graphs/worship-service> AS ?graphTwo)
+          }
+          ${subject} org:hasSite ?site.
+          ?site ?pSite ?oSite.
+          {
+            ?site org:siteAddress ?contact.
+            ?contact ?pContact ?oContact.
+            ?contact locn:address ?addressContact.
+            ?addressContact ?pAddressContact ?oAddressContact.
+          }
+          UNION
+          {
+            ?site org:siteAddress ?contact.
+            ?contact ?pContact ?oContact.
+          }
+          UNION
+          {
+            ?site organisatie:bestaatUit ?address.
+            ?address ?pAddress ?oAddress.
+          }        
+          BIND(IRI(CONCAT("http://mu.semte.ch/graphs/organizations/", ?uuid)) AS ?graph)
```

Comment:
**Question:** Is this line necessary? The `?graph` variable is not used as far as I see. (In case the variable is meant to be used the graph URI should be corrected.)